races/chicken.md

```diff
@@ -1,3 +1,45 @@
 ---
-wrapper: "../../../js/components/wiki/StubArticle"
+wrapper: "../../../js/components/wiki/BasicArticle"
+prependJs: 
+  - "import Grid from '@material-ui/core/Grid';"
+title: Chicken
+description: Bawk.
 ---
+You know what a chicken is, right?  This is a chicken.  There is no good reason to be a chicken.  
+
+### Starting Stats
```

Comment:
I think the header should have the whole word "Statistics" instead of "Stats"

Reply:
```suggestion
### Starting Statistics
```

Makes sense. I'll put in a change for Android for the same thing.